ukbb_qc/release/prepare_vcf_data_release.py

```diff
@@ -450,22 +462,26 @@ def main(args):
                 {"vep": {"Description": hl.eval(ht.vep_csq_header)}}
             )
 
-            # NOTE: Correcting rsid here because this was discovered after release HT was created
+            # Re-add rsid to HT and reformat
             logger.info("Reformatting rsid...")
             # dbsnp might have multiple identifiers for one variant
             # thus, rsid is a set annotation, starting with version b154 for dbsnp resource:
             # https://github.com/broadinstitute/gnomad_methods/blob/master/gnomad/resources/grch38/reference_data.py#L136
             # `export_vcf` expects this field to be a string, and vcf specs
             # say this field may be delimited by a semi-colon:
             # https://samtools.github.io/hts-specs/VCFv4.2.pdf
+            dbsnp_ht = dbsnp.ht().select("rsid")
+            ht = ht.annotate(rsid=dbsnp_ht[ht.key].rsid)
             ht = ht.annotate(rsid=hl.str(";").join(ht.rsid))
 
             logger.info(
                 "Selecting relevant fields for VCF export and checkpointing HT..."
             )
             ht = ht.select("info", "filters", "rsid", "qual")
             ht.write(
-                release_vcf_ht_path(*tranche_data), overwrite=args.overwrite,
+                # release_vcf_ht_path(*tranche_data), overwrite=args.overwrite,
+                "gs://broad-ukbb/broad.freeze_7/release/ht/broad.freeze_7.release.vcf.reexport.ht",
```

Comment:
Not really needed, but maybe rename with `ukb_official_export` or something like that instead of `reexport`?